### PR TITLE
New oncoprint property to configure tracks that are shown by default (oncoprintjs)

### PR DESCRIPTION
**Pull request by dianab0**
Change necessary for https://github.com/cBioPortal/cbioportal-frontend/pull/4152

Added a new value for the initial gaps value (enabled/disabled) for the clinical tracks.

### CODE REVIEW

File: src/js/oncoprintmodel.ts [hunk 1]
```diff
@@ -87,6 +87,7 @@ export type UserTrackSpec<D> = {
     custom_track_options?:CustomTrackOption[];
     $track_info_tooltip_elt?:JQuery;
     track_can_show_gaps?:boolean;
+    init_show_gaps?:boolean
```
Comment (pvannierop) [inline src/js/oncoprintmodel.ts:90]: Btter rename to `show_gaps_on_init`

File: src/js/oncoprintmodel.ts [hunk 2]
```diff
@@ -1153,7 +1154,7 @@ export default class OncoprintModel {
         }
 
         this.track_can_show_gaps[track_id] = ifndef(params.track_can_show_gaps, false);
-        this.track_show_gaps[track_id] = false;
+        this.track_show_gaps[track_id] = (params.init_show_gaps === true) ? true : false;
```
Comment (pvannierop) [inline src/js/oncoprintmodel.ts:1157]: Please simplify to:
```
this.track_show_gaps[track_id] = params.init_show_gaps;
```
:)

**Review by pvannierop — CHANGES_REQUESTED** [review]

**Review by adamabeshouse — APPROVED** [review]
looks good after small changes

File: src/js/oncoprintmodel.ts (hunk 2 above)
Comment (pvannierop) [reply src/js/oncoprintmodel.ts:1157]: Ah I did not notice that convention. Good catch.

Comment (dianab0) [reply src/js/oncoprintmodel.ts]: Done

File: src/js/oncoprintmodel.ts (hunk 1 above)
Comment (dianab0) [reply src/js/oncoprintmodel.ts]: Done

**Review by pvannierop — APPROVED** [review]

**Pull request merged**